src/amoc.erl

```diff
@@ -1,7 +1,8 @@
-%%==============================================================================
-%% Copyright 2023 Erlang Solutions Ltd.
-%% Licensed under the Apache License, Version 2.0 (see LICENSE file)
-%%==============================================================================
+%% @copyright 2023 Erlang Solutions Ltd.
+%% @doc API module for running locally, in a non-distributed environment
```

Comment:
Maybe we could use `@moduledoc` attribute? https://hexdocs.pm/elixir/1.12/writing-documentation.html#module-attributes If you do so, then please change it everywhere.

Reply:
That's for elixir, for erlang it doesn't work 🥲